name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/erstebank/Kauf01.txt

```diff
@@ -0,0 +1,36 @@
+PDF Autor: ''
```

Comment:
PDFBox Version: 1.8.16
Portfolio Performance Version: 0.59.1
-----------------------------------------
Sparkasse Neuhofen Bank Marktplatz 18 T 05 0100 Firmensitz: Neuhofen an der Krems
Aktiengesellschaft 4501 Neuhofen an der Krems F 05 0100-942300 Landesgericht Linz
www.sparkasse.at/neuhofen  UID: ATU22722409, FN 123543 g
Seite 1/2   BIC: SPNKAT21XXX
WERTPAPIER WERTPAPIERDEPOT
Kauf Marktplatz 00088-061312
 
lautend auf ELIAS PALNIK
LASTENSTRAßE 23, TOP. 17.10, 4020 LINZ
Herr 220/0000/KAM/A
Elias Palnik
Lastenstraße 23, Top. 17.10
4020 Linz
WERTPAPIER
Kauf Marktplatz
Ihr Auftrag vom 08.09.2022, 14:10 Uhr Auftragsnummer 12003580
DE0008404005 ALLIANZ SE Kupon: 29
VINK.NAMENS-AKTIEN O.N.
Limit: Bestens beratungsfreies Geschäft
Nominale/Stück Kurs  Kurswert
 
Marktplatz: XETRA FFT Ausführungszeit: 14:10:34 Schlusstag: 08.09.2022
10,00 STK 166,86 EUR  1.668,60 EUR
 Einmalige Kosten 48,78 EUR
 
Zu Lasten 00001-152198 12.09.2022 1.717,38 EUR
Verrechnungskonto Valuta Endbetrag gesamt
Wertpapierrechnung, Verwahrort: FRANKFURT
KGSt - Anschaffungskurs berechnet
Vorbehaltl. Eingang bzw. nachträgl. Storni durch Partner.
Kosten im Detail
Die angegebenen Kosten und Gebühren wurden anhand des Abrechnungskurses berechnet.
Die %-Werte beziehen sich auf den errechneten Kurswert von 1.668,60 Euro.
 
Aufgrund der Rundungsregeln sind geringfügige Abweichungen zwischen absoluten Beträgen und
Prozentwerten möglich.
 
Sparkasse Neuhofen Bank Marktplatz 18 T 05 0100 Firmensitz: Neuhofen an der Krems
Aktiengesellschaft 4501 Neuhofen an der Krems F 05 0100-942300 Landesgericht Linz
www.sparkasse.at/neuhofen  UID: ATU22722409, FN 123543 g
Seite 2/2   BIC: SPNKAT21XXX
WERTPAPIER WERTPAPIERDEPOT
Kauf Marktplatz 00088-061312
 
lautend auf ELIAS PALNIK
LASTENSTRAßE 23, TOP. 17.10, 4020 LINZ
Ihr Auftrag vom 08.09.2022, 14:10 Uhr Auftragsnummer 12003580
(1) Dienstleistungskosten
 
Einmalig, bei Kauf
   
Eigene Kosten
Transaktionsgebühr EUR 35,00  
Bearbeitungsgebühr EUR 5,00  
   
Fremde Kosten
Transaktionsgebühr EUR 8,78  
 
Summe der Dienstleistungskosten EUR 48,78 2,92 %
 
Zuwendungen
 
Unser Institut erhielt für diese Transaktion keine einmaligen Zuwendungen für dieses Wertpapier.
 
(2) Produktkosten
 
Es fielen für die Transaktion keine produktbezogenen Kosten der EmittentIn an.
 
(3) Gesamtkosten
 
Summe aller Kosten EUR 48,78 2,92 %
 
 
Begriffserklärung
 
(1) Dienstleistungskosten können beispielsweise bei Kauf, Verkauf oder Lagerung eines Wertpapiers anfallen.
Der Begriff eigene Kosten umfasst alle Kosten, die bei Ihrer Bank anfallen, z. B. für die Ausführung der Order
oder die Umrechnung von einer Fremdwährung in Euro.
Mit fremden Kosten sind alle Kosten gemeint, die außerhalb Ihrer Bank anfallen, also z. B. Handelsgebühren an
der Börse.
Zuwendungen sind Zahlungen Dritter an Ihre Bank (wie z. B. Bestandsprovisionen). Die Sparkassen Gruppe
verwendet diese für qualitätsverbessernde Maßnahmen im Wertpapierbereich.
 
(2) Produktkosten sind die Kosten einer EmittentIn, die z. B. für die Entwicklung oder Vermarktung eines
Wertpapieres bereits in den Kaufpreis eingerechnet wurden. Diese Kosten fallen beim Kauf oder Verkauf eines
Wertpapieres nicht extra an, sind aber zur Transparenz am Kostenausweis angeführt.
 
Neuhofen an der Krems, am 08.09.2022
Wir danken für Ihren Auftrag

Reply:
Hallo @DaaPalnik 
Vielen Dank für den PDF-Debug.
Bitte erstelle ein PDF-Debug und poste diesen [hier](https://forum.portfolio-performance.info/t/pdf-import-von-erste-bank-brokerjet-sparkassengruppe-osterreich/15433).
Wie das funktioniert, schaust du dir am besten im Video-Tutorial an.

Video-Tutorial:
[PDF-Dokumente für die Fehlersuche extrahieren](https://forum.portfolio-performance.info/t/buchungen-aus-pdf-dateien-importieren/38#tutorial-extrahieren)

Gruß
Alex

Reply:
@DaaPalnik 
ich habe deinen PDF-Debug hinzugefügt. 
#2973